sync/sync-api/src/main/java/com/duckduckgo/sync/api/engine/Models.kt

```diff
@@ -33,16 +34,19 @@ data class SyncChangesRequest(val type: SyncableType, val jsonString: String, va
 data class SyncChangesResponse(val type: SyncableType, val jsonString: String) {
 
     companion object {
-        fun empty(): SyncChangesResponse {
-            return SyncChangesResponse(BOOKMARKS, "")
+        fun empty(type: SyncableType): SyncChangesResponse {
+            return SyncChangesResponse(type, "")
         }
     }
 }
 
+// TODO: document api
 enum class SyncableType(val field: String) {
     BOOKMARKS("bookmarks"),
+    CREDENTIALS("credentials"),
 }
 
+// TODO: document api, when is it expected each case?
```

Comment:
do it?

Reply:
That was for me to create a task in Asana, not to fix it as part of this PR.

Changed comment to refer to asana instead 👍